k8s/presets/presets.go

```diff
@@ -10,9 +10,38 @@ import (
 	"github.com/smartcontractkit/chainlink-testing-framework/k8s/pkg/helm/reorg"
 )
 
+var BaseToml = `[Log]
```

Comment:
Are we able to use the config-builder logic we use for docker envs and just `.String()` it instead?

Reply:
Not unless we want to import chainlink into this repository, which we don't.